bedrock/firefox/templates/firefox/os/tv.html

```diff
@@ -0,0 +1,83 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "/firefox/base-resp.html" %}
+
+{% block page_title_prefix %}{{ _('Firefox OS — ') }}{% endblock %}
+{% block page_title %}{{ _('Beyond smartphones — Bringing the power of the Web to Panasonic TVs') }}{% endblock %}
+{% block page_desc %}{{ _('Built entirely using HTML5 and other open Web standards, Firefox OS is free from the rules and restrictions of existing proprietary platforms.') }}{% endblock %}
+{% block body_id %}firefox-os-tv{% endblock %}
+{% block body_class %}firefox-os-tv blueprint{% endblock %}
+
+
+{% block site_css %}
+  {{ css('firefox_os_tv') }}
+{% endblock %}
+
+{% block site_header_nav %}{% endblock %}
+
+{% block site_header_logo %}
+  <h2><a href="{{ url('firefox.os.index') }}">
+  {{ high_res_img('img/firefox/os/devices/tv/logo.png', {'alt': 'Firefox OS', 'width': '221', 'height': '67'}) }}</a></h2>
+{% endblock %}
+
+{% block content %}
+
+<div class="main-feature">
+  <h1>{{_('Firefox OS comes to TVs')}}</h1>
+  <h2>{{ _('We’re bringing the power of the Web to Panasonic televisions later this year.') }}</h2>
+</div>
+
+<div id="main-content">
+
+  <div class="pager pager-auto-init pager-with-nav pager-with-tabs pager-no-history" id="mozilla-pager-1">
+
+    <div class="tv">
+      <div class="pager-content" id="mozilla-pager-tv-pages">
+        <div id="tv1" class="pager-page">
+          {{ high_res_img('img/firefox/os/devices/tv/screen1.jpg', {'alt': _('Screenshot'), 'width': '796', 'height': '448'}) }}
```

Comment:
This alt text isn't especially useful. If we can't describe what's on the screen we should just have empty alt.